readthedocs/api/v2/permissions.py

```diff
@@ -83,3 +84,33 @@ def has_object_permission(self, request, view, obj):
             request.method in permissions.SAFE_METHODS or
             (request.user and request.user.is_staff)
         )
+
+
+class FooterHTMLRestrictedPermission(permissions.BasePermission):
```

Comment:
"Restricted" is not a good name for this class. It should be more descriptive about what is the restriction.

Reply:
All classes in this file are called like that. In .com I called this `IsAuthorizedToViewVersion`, that's how most of the permissions of DRF are called.

Reply:
`IsAuthorizedToViewVersion` is clear.

My point is that `Restricted` does not say _why_ it's restricted or _when_ it's allowed.